can-map-define.js

```diff
@@ -9,6 +9,8 @@ var CanMap = require('can-map');
 var compute = require('can-compute');
 var canReflect = require('can-reflect');
 var ObservationRecorder = require('can-observation-recorder');
+var Resolver = require("can-simple-observable/resolver/resolver");
```

Comment:
I think `can-simple-observable` needs to be added to `package.json`

Reply:
I've added it, it's strange that it worked and tests passed without it.